CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.9.0] - 2022-02-03
+
```

Comment:
Adding a changelog would be good :grimacing:

Reply:
oh shit, forgot about it